### DISCUSSION TITLE
Issue with Large Prints on V400

### DISCUSSION THREAD

**Discussion by Octoross** [blackfyre/flsun.community#75] [Q&A]
Hello everyone,
I am having issues printing large items with my V400 and I am spending a LOT of filament on failed prints at the moment.  I don't seem to be able to print at default CURA speeds for parts with infill in them.  after several layers (~50) I hear the nozzle grinding on the infill and it gets the printer completely out of alignment.  I tried reducing speed to ~200mms on the infill, activating z-hop when retracted, reducing acceleration to 5000mms, I just never know when it's going to skip and start messing up the print...

At one point, I even had the printer randomly dig in the bed and then print not parallel to the bed anymore - this seems to indicate one of the axis skipped even though there was no mechanical impact on the nozzle at that time....  

Is there a new profile to mitigate this?  If this is related to belt tension, how should the tension be set on the belts and how do I test to see if they are all ok?  Also, z-axis printing always appears to be a bit more than it should by about 1%.  should I calibrate z steps or is there something else I'm missing?  

I calibrated the bed with a hot bed to mitigate movements due to heat.  First layer has been difficult to set - needs to be quite a bit higher than the paper test - but appears nice now and sticks well to the bed.  I'm printing PLA @ 210 with bed at 60 degrees.

All the presliced models on the usb key print fine at full speed...

Thanks

**Comment by x24bvitaldigital** [comment]
We have 3 Flsun V400's and are seeing similar behavior.  Is there further discussion on this?

Slowing down to 200 mm/s does help but does not solve the issue.

**Comment by Aviador75** [comment]
Hola la mia tambien me roza en las impresiones y tengo que parar la impresión,  pero con los modelos que trae la impresora  no tengo ese problema  los hace perfectamente y con calidad suprema, creo que puede ser un ajuste en cura que no estoy poniendo bien.

**Comment by VRENA** [comment]
I purchased a V400 a week ago, and I am having the same issues.  Is there no response for this?  I tried tech support, and they gave me all of the same fixes that are available online / through their videos.  I have done the following:

Updated firmware
Changed z-hop
lowered print speed
calibrated z height multiple times with the paper trick (checking for drag etc)
during printing I tried doing the fine tuning to change the z-height, no change still scrapes
calibrated bed
bought accelerometer and calibrated
tried different filament (even though the head is scraping, which seems to be a calibration issue)
did the belt tension steps (printed the tension scale and tried a few different tension settings)

What is strange is that there are some things that print just fine, but what I am having issues with is anything that is somewhat large, or anything that is in a circle shape.  It just scrapes.
